package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "todo",
+  "version": "1.0.0",
+  "description": "",
+  "private": true,
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack",
```

Comment:
- [ ] Your webpack is being compiled with errors. Please fix this:
![Screenshot from 2022-03-22 22-34-43](https://user-images.githubusercontent.com/62705947/159580374-cedb9412-cf4d-4f99-ab0e-ad1ab1fab44d.png)